src/jit/gentree.h

```diff
@@ -771,8 +770,8 @@ struct GenTree
 // TODO-Cleanup: Currently, GTF_REG_BIRTH is used only by stackfp
 //         We should consider using it more generally for VAR_BIRTH, instead of
 //         GTF_VAR_DEF && !GTF_VAR_USEASG
-#define GTF_REG_BIRTH       0x04000000 // GT_REG_VAR -- enregistered variable born here
-#define GTF_VAR_DEATH       0x02000000 // GT_LCL_VAR, GT_REG_VAR -- variable dies here (last use)
+#define GTF_REG_BIRTH       0x04000000 // GT_LCL_VAR, -- enregistered variable born here
```

Comment:
@CarolEidt Is this comment change valid? I don't think GTF_REG_BIRTH is used at all now.

Reply:
Right - I don't think it's correct, but I think it's reasonable to eliminate it with #18196